import.sh

```diff
@@ -2,17 +2,20 @@
 
 set -eu
 
-rm -rf *.c internal/*
+rm -rf internal/*
+find . -type l | xargs rm
```

Comment:
Done.

On Thu, Apr 28, 2016 at 12:16 PM, Ben Darnell <notifications@github.com>
wrote:

> In import.sh
> <https://github.com/cockroachdb/c-jemalloc/pull/4#discussion_r61456889>:
>
> > @@ -2,17 +2,20 @@
> >
> >  set -eu
> >
> > -rm -rf *.c internal/*
> > +rm -rf internal/*
> > +find . -type l | xargs rm
>
> We don't want to delete any symlinks that may exist in the .git
> directory. (not sure if that ever happens "naturally", but in our main repo
> we put symlinks in .git/hooks)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/c-jemalloc/pull/4/files/b6470cb2910dbb4a91157b7932492073c76ab44a#r61456889>
>